### ISSUE TITLE
feat(world-model): qualify context- and horizon-aware model arbitration without turning disagreement into evidence

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symthaea#640]
## Purpose

Define Symthaea's first **predictive-model arbitration** theorem so multiple world/dynamics/generative models can coexist, compete, abstain, specialize and be revised using fresh evidence without treating model disagreement as physical evidence, winner selection as truth, ensemble member count as independent support, or one historically strong model as universally authoritative.

Symthaea already has or plans materially different predictive mechanisms:

```text
persistence / no-change
constant metric velocity
CfC / liquid dynamics
object-factorized learned dynamics
explicit geometry / rigid transforms
photometric renderer models
future physics/contact models
```

The architecture should preserve that diversity rather than prematurely forcing one universal world-model implementation.

Intended ordering:

```text
qualified physical belief
        +
#594 transition evidence
        +
#598 candidate physical dynamics models
        +
#580 generative observation models
        +
#631 currentness/change context
        +
#633 multimodal fresh evidence
        ↓
THIS ISSUE: model hypothesis set + arbitration/evaluation
        ↓
context/horizon-specific predictive distribution
        ↓
#607 active model-discriminating perception
        +
#635 robust belief-space planning
        ↓
fresh execution/observation evidence
        ↓
model score/update/revision
```

This issue does not itself design a new predictive architecture. It defines how candidate models are compared and consumed without epistemic laundering.

## Core non-equivalences

Freeze explicitly:

```text
model prediction != observation
model disagreement != physical evidence
ensemble variance != calibrated epistemic uncertainty by default
number of models != independent sample count
multiple random seeds != independent scientific theories
same training data + different initialization != independent evidence roots
best training loss != best deployed model
lowest one-step residual != true dynamics
best RGB prediction != best physical state prediction
winner at horizon h1 != winner at horizon h10
winner in context C != universal winner
model weight != physical probability unless exact semantics justify it
posterior model weight != fresh observation
model selection != truth
model consensus != truth
all models agreeing != world proof
all models disagreeing != sensor failure
OOD score != automatic rejection truth
robust-loss weight != model correctness
planning value != predictive validity
planner exploiting model != evidence that action is safe
```

Most importantly:

```text
models are hypotheses about transition/observation structure
        not sources of physical-world evidence
```

## Candidate model identity and scope

Every candidate should have explicit semantics, conceptually:

```text
WorldModelHypothesisRefV1
├── model namespace / local id / generation
├── model family / state-action schema
├── trained artifact / parameter digest
├── training lineage ref
├── prediction semantics
├── declared applicability envelope
├── supported horizon/action/context domain
└── model_digest
```

Changing weights, training data, architecture, feature semantics, calibration or model state is a new model revision/generation where appropriate.

Do not identify a model only by a human label such as `cfc` or `physics`.

## Prediction schemas must be comparable before scoring

Two models can only compete directly when they predict the same declared target/event under compatible semantics.

Examples:

```text
model A predicts object SE(3) pose
model B predicts RGB image
```

cannot be ranked by one undifferentiated loss.

Likewise:

```text
optical-z depth
    !=
Euclidean range
```

unless an explicit qualified conversion is part of the scoring profile.

Use typed prediction targets such as:

```text
ObjectPoseAtHorizonV1
SurfaceStateAtHorizonV1
PredictedObservationV1
ActionOutcomeEventV1
```

and score only compatible predictions.

## Arbitration is context-specific

Do not maintain one global model leaderboard as the sole runtime choice.

A model may be good for:

- static objects;
- low-speed rigid motion;
- short horizons;
- a particular tool/object family;
- one sensor/appearance regime;

and fail elsewhere.

Conceptually:

```text
ModelApplicabilityContextV1
├── current physical state / object class where relevant
├── action family / magnitude
├── horizon / dt
├── interaction/contact regime
├── sensor/currentness state
├── embodiment/controller profile
├── OOD diagnostics
└── context_digest
```

Model performance/weights are conditioned on this exact context or a declared generalization class.

```text
best globally on average
        !=
best here now
```

## Horizon is first-class

Prediction uncertainty and model ranking can change with rollout horizon.

Current 2026 world-model work explicitly emphasizes horizon-calibrated uncertainty and compounding rollout error.

Therefore model evaluation must measure:

```text
score(model, context, horizon)
```

not one timeless score.

A short-horizon CfC model may outperform constant velocity at 200 ms yet become less reliable at 5 s; a simple persistence model may remain the safest fallback in a nearly static regime.

Allow a model's validity horizon to be bounded:

```text
UsefulThrough(h <= H)
TooUncertainBeyond(H)
```

rather than extrapolating unchanged confidence.

## Fresh evidence owns model scoring

A model earns predictive support only when a prediction is committed **before** fresh outcome evidence and later compared against that fresh evidence.

Preferred sequence:

```text
Belief_t + Action_t
        ↓
commit prediction P(model, horizon)
        ↓
action execution / time advances
        ↓
fresh observation O_{t+h}
        ↓
qualified scoring residual / proper score
        ↓
model performance update
```

Never score a model using the same outcome that was used to construct/fine-tune the prediction without explicitly labeling that as training, not held-out predictive evidence.

## Prediction commitment / custody

For confirmatory model comparison, preserve:

- exact model revision;
- exact input belief/action/context;
- prediction digest;
- issue/commit time or phase;
- later fresh observation identity;
- scoring profile.

This prevents hindsight leakage where a model is regenerated after seeing the outcome and then called predictive.

For controlled experiments, use the repository's existing evidence/research-custody principles where practical.

## Proper scores and physical metrics

Do not select models only by visual plausibility or latent loss.

For probabilistic predictions, use proper scoring metrics where event semantics permit:

- log score/NLL;
- Brier score;
- calibration/coverage;
- CRPS or appropriate distributional score;

alongside physical task metrics:

- translation/rotation error;
- surface displacement;
- contact/state transition correctness;
- horizon-dependent drift;
- rendered depth/RGB only for observation models.

A model can win photometric metrics while losing physical dynamics metrics; preserve both.

## Model disagreement is diagnostic, not evidence

For an ensemble/model set:

```text
{P_1, P_2, ..., P_N}
```

disagreement can indicate:

- epistemic uncertainty;
- different inductive biases;
- OOD input;
- multimodal future possibilities;
- one or more broken models.

But:

```text
variance(P_i)
        !=
physical evidence about the world
```

and not automatically calibrated uncertainty.

Use a typed diagnostic:

```text
ModelDisagreementAssessmentV1
├── target/horizon/context
├── candidate predictions
├── disagreement metric
├── training/shared-root diagnostics
├── calibration state
└── disagreement_digest
```

The disagreement may inform #607 information gathering or #635 robust planning but cannot update physical belief by itself.

## Ensemble members share training ancestry

A bootstrapped ensemble, random-seed ensemble or mixture of experts may provide useful uncertainty proxies.

Do not call members statistically independent because their parameter initializations differ.

Preserve shared training roots:

```text
same dataset / same simulator / same labels
        -> shared epistemic lineage
```

Ensemble size must not be interpreted as evidence count.

```text
20 models trained on same 1,000 transitions
        !=
20,000 physical transitions
```

## Model weights are not observation weights

An arbitration system may maintain context-specific weights or probabilities over model hypotheses.

Those weights describe predictive-model plausibility/performance under the declared arbitration semantics.

They are not sensor likelihood weights and do not multiply physical evidence count.

Conceptually:

```text
ModelArbitrationStateV1
├── model hypothesis refs
├── context/horizon scope
├── historical predictive score summaries
├── model applicability/OOD states
├── calibrated model-selection/mixing weights if justified
├── unresolved disagreement
└── arbitration_digest
```

If a Bayesian model-probability interpretation is claimed, state the prior, likelihood and model-set assumptions exactly.

Otherwise call them weights/scores, not posterior probabilities.

## No forced winner

Valid arbitration results include:

```text
Select(model A)
Mixture(models A,B,...)
RobustSet(models A,B,...)
NoAdequateModel
ModelsIndistinguishable
ModelsConflict
OutOfQualifiedDomain
```

Do not force winner-take-all merely because an API expects one model ID.

A robust planner may evaluate candidate actions across a set of plausible models instead of collapsing uncertainty prematurely.

## Mixtures must preserve multimodality

Averaging incompatible model predictions can create a physically impossible mean.

Example:

```text
model A: object moves left
model B: object moves right
mean: object stays centered
```

The mean may have low physical plausibility.

Where futures are multimodal, preserve a mixture/hypothesis set or another calibrated multimodal representation rather than averaging state vectors blindly.

## Model class mismatch / misspecification

A candidate set may omit the true dynamics class.

If every model performs poorly or residual structure violates assumptions:

```text
NoAdequateModel / ModelClassMisspecification
```

must be representable.

Do not normalize bad weights to 100% and then report one bad model with false certainty.

This state can trigger:

- shorter horizon;
- conservative fallback;
- active experiment/information gathering;
- model-family expansion proposal;
- human/research review.

## Active model discrimination

#607 can target uncertainty about **which model predicts reality better**.

A candidate action/view may be valuable because models make sharply different predictions there.

Conceptually:

```text
models A,B disagree on action consequence
        ↓
choose safe discriminating intervention/view
        ↓
fresh outcome
        ↓
update model arbitration
```

But raw disagreement alone is not sufficient to justify a physical intervention; #635 risk/utility and the authority boundary still apply.

The experiment must be safe/authorized and expected to produce distinguishable observations.

## Relationship to self-improvement

Model arbitration can support bounded recursive improvement without allowing a new model to self-certify.

Preferred promotion path:

```text
candidate model proposal
        ↓
qualification / frozen evaluation profile
        ↓
held-out predictive contest against current models
        ↓
replication / OOD tests where required
        ↓
promotion for a declared context/horizon
```

Never:

```text
new model generated by system
    -> system says it is better
    -> immediate canonical replacement
```

A candidate can remain experimental while runtime arbitration gathers evidence.

## Training/update and evaluation separation

For learned models preserve distinct:

```text
fit data
calibration data
evaluation data
online deployment outcomes
```

Online adaptation may update a model, but the updated model revision cannot claim predictive credit for the same outcome used to update it.

The next fresh outcomes evaluate the new revision.

## Catastrophic switching / hysteresis

A noisy one-step score should not cause unstable rapid switching between models unless that behavior is explicitly desirable and qualified.

Candidate arbitration policies may include:

- bounded evidence accumulation;
- hysteresis/change threshold;
- Bayesian filtering over model state;
- context-gated experts;
- robust mixture;

and should be compared experimentally.

Do not hide switching behavior inside incidental floating-point ties or model enumeration order.

## Model currentness / environment change

#631 world changes can invalidate historical model performance.

A dynamics model trained before payload, friction, tool or environment changes may no longer be current.

Model applicability should therefore be revisable by context/currentness rather than permanently accumulating global reputation.

```text
historically strong model
        !=
currently valid model
```

## Planning boundary

#635 can consume:

- selected model where evidence strongly supports one;
- a mixture/hypothesis set;
- worst-case/robust subset;
- horizon-specific predictive uncertainty;
- `NoAdequateModel`.

The planner must not cherry-pick whichever model makes an action look safest/cheapest.

Bind the exact arbitration state/profile into the plan identity.

## Qualification fixtures

Use hidden true dynamics only for evaluator scoring.

Include at least:

1. static object where persistence is correct;
2. constant-velocity motion;
3. action-conditioned rigid push;
4. nonlinear motion where CfC candidate may help;
5. short-horizon vs long-horizon ranking reversal;
6. model good in one object/action class, poor in another;
7. all models wrong / omitted dynamics class;
8. two genuinely multimodal future branches;
9. mean-of-two-models physically implausible fixture;
10. ensemble members sharing one training dataset;
11. different architectures sharing one dataset;
12. genuinely disjoint training lineages where available;
13. OOD action magnitude;
14. environment/currentness shift;
15. model update on outcome then attempted self-score leakage;
16. prediction committed before fresh outcome;
17. post-hoc regenerated prediction negative control;
18. one-step residual winner with poor long-horizon rollout;
19. photometric winner with wrong physical pose;
20. model-disagreement active experiment;
21. unsafe discriminating action rejected by planner/authority;
22. repeated score updates from same observation rejected/deduplicated;
23. noisy score causing potential rapid switching;
24. model enumeration-order permutation.

## Baselines

Compare at least:

```text
R0: fixed single model
R1: global best historical model
R2: lowest current residual / winner-take-all
R3: simple ensemble mean
R4: calibrated context/horizon-aware model mixture/selection
R5: R4 + explicit OOD/misspecification/robust planning interface
```

Where feasible include a simple persistence/constant-velocity baseline in every contest so learned complexity must earn its place.

### Predictive performance
- proper scores / physical error by target;
- error vs horizon;
- error by context/action/object class;
- long-horizon failure rate;
- multimodal prediction quality.

### Calibration / arbitration
- model-selection accuracy when evaluator model class is known;
- weight/reliability calibration where claimed;
- regret vs evaluator-best model/mixture;
- `NoAdequateModel` precision/recall;
- OOD/misspecification detection.

### Stability
- unnecessary model switch rate;
- recovery time after true regime change;
- sensitivity to one anomalous observation;
- enumeration-order invariance.

### Epistemic integrity
- model/ensemble count does not change physical evidence support;
- same fresh outcome cannot be scored repeatedly as independent evidence;
- shared training ancestry remains visible;
- model disagreement alone changes no physical belief;
- post-hoc prediction cannot receive prospective credit.

### Downstream utility
- #635 task/risk performance with arbitration vs fixed model;
- #607 information efficiency for model-discriminating actions;
- failure under planner exploitation of weak models.

### Systems
- arbitration latency;
- active model count/budget;
- prediction-cache/replay determinism;
- bounded memory.

## Canonical identities

Separate at least:

```text
world_model_hypothesis_digest
    = model family/schema
    + exact parameters/artifact
    + training lineage
    + prediction/applicability semantics

model_prediction_digest
    = exact model revision
    + input belief/action/context
    + horizon
    + canonical prediction

model_score_record_digest
    = precommitted prediction
    + fresh outcome evidence
    + scoring profile
    + canonical score/residual

model_arbitration_profile_digest
    = candidate model set rules
    + context/horizon partition
    + scoring/calibration/update rule
    + OOD/misspecification policy
    + mixture/selection/switching rule
    + budgets

model_arbitration_state_digest
    = exact model revisions
    + accepted score records
    + context/horizon state
    + canonical weights/selection/disagreement status
```

No model/arbitration digest grants observation, truth, safety or action authority.

## Research grounding

Current Physical-AI world-model research treats uncertainty and model bias as central closed-loop problems. Recent surveys emphasize probabilistic ensembles, stochastic state models, rollout-horizon management and planner exploitation as key design tensions. QUEST uses ensemble dynamics to guide exploration versus conservative planning under OOD uncertainty. HAUWM explicitly trains predictive uncertainty to scale with temporal horizon. Related work on world-model verification highlights a further problem: exploration rules based only on a model's own uncertainty can inherit the model's weaknesses in underexplored regions.

Symthaea should add a stronger epistemic arbitration layer: multiple candidate mechanisms remain hypotheses, fresh outcomes score precommitted predictions, model diversity cannot manufacture physical evidence, and no candidate self-certifies its own promotion.

## Kill conditions

V1 fails if an implementation:

- treats model/ensemble disagreement as direct world evidence;
- interprets ensemble member count or random seeds as independent physical support;
- chooses models by incompatible target metrics;
- maintains one global winner without context/horizon scope while making scoped claims;
- scores a prediction after adapting/regenerating it using the outcome;
- lets one fresh observation produce repeated independent model-score updates;
- averages multimodal future hypotheses into physically implausible mean state without explicit semantics;
- normalizes all-bad model weights into false certainty instead of permitting `NoAdequateModel`;
- lets historical model reputation override current OOD/currentness evidence;
- lets a planner cherry-pick the model that makes an action look best;
- allows a self-generated candidate model to promote itself without held-out qualification;
- uses hidden evaluator future truth at runtime.

## Exit gate

V1 qualifies only if held-out experiments establish that:

1. candidate models have exact target/context/horizon semantics and are compared only on compatible predictions;
2. fresh evidence scores precommitted predictions without hindsight leakage;
3. model disagreement/ensemble size cannot change physical evidence support by itself;
4. context- and horizon-aware arbitration improves predictive calibration/regret over fixed/global/winner-take-all baselines;
5. multimodal hypotheses and `NoAdequateModel` remain representable instead of forced averaging/selection;
6. shared training lineage is visible and ensemble uncertainty is empirically calibrated before stronger claims;
7. model-switching and regime-change behavior are measured rather than incidental;
8. #607/#635 can use arbitration uncertainty for safe information gathering/robust planning without self-evidence;
9. candidate model promotion is qualification-based and cannot be self-certified;
10. no model/arbitration result grants truth, safety certification or action authority by construction.
